ministryofjustice/correspondence_tool_staff
# CT-2966 Use pre tag and tidy style

## Description
Format address as it was entered in the boxes for both instances on case details page, and make labels align

## Self-review checklist


### Screenshots
Was:
![image](https://user-images.githubusercontent.com/22935203/89808314-e203c600-db31-11ea-90cd-4f8c7c5b6f7d.png)


Is:
![image](https://user-images.githubusercontent.com/22935203/89808263-cc8e9c00-db31-11ea-9f0d-857d5963aeca.png)


### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-2966

### Deployment
n/a

### Manual testing instructions
Add an Offender SAR case and put line breaks in address of subject. Then choose 3rd party as recipient and add linebreaks there too, and some spaced on same line to test. 2x Addresses on case details page should show as entered in the box. Check label aligns to top also.
